src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -985,122 +1003,155 @@ <h2 class="heading-medium">
   </thead>
   <tbody>
 
-  <tr>
-    <td>
-      <div>
-        Support required for a hearing
-      </div>
-    </td>
-    <td>
-      <div>
-        {% for support in hearingContent.supportRequired %}
-        {{ support }}<br/>
-        {% endfor %}
-      </div>
-    </td>
-  </tr>
+  {% if hearingContent.supportRequired is not empty %}
+    <tr>
+      <td>
+        <div>
+          Support required for a hearing
```

Comment:
I guess this should always appear as this is the default question to everyone

Reply:
now its optional